.github/workflows/ci.yml

```diff
@@ -3,7 +3,6 @@ name: CI
 on:
   pull_request:
     branches:
-      - develop
```

Comment:
Optional, but making a PR into a PR (like this one) is handy.

Reply:
Can't you just make suggested edits on the PR?

Reply:
Yes, if you look at the history of your PR you will see a force push from me that seems to do nothing. It turned out to a bit less simple than I expected to fix (I didn’t catch the name issue at first), so I undid that and moved to a PR.

Reply:
Oh, suggested edits, no, you can’t suggest an edit for something more than a few lines away from a change. And you can’t see if the edit passed CI. :)